src/persist/src/indexed/mod.rs

```diff
@@ -1367,8 +1352,22 @@ mod tests {
             i.write(vec![(id, updates.clone())], handle)
         })?;
 
+        // Sanity check that the data is all unconsolidated in unsealed and none of
```

Comment:
nit: it's strange for the test to be actually asserting that we want them unconsolidated. it's worth asserting that it's in unsealed, so I think my issue is just with the phrasing of this comment. let's remove the bit here about them being unconsolidated (and if you like, put something above the assert_eq on line 1358 to the effect of "// Note that unsealed does not consolidate")